### DISCUSSION TITLE
Unable to Visualize by log.file.path in 2.3.110

### DISCUSSION THREAD

**Discussion by mciantar** [Security-Onion-Solutions/securityonion#7537] [Unsupported Versions]
I have some visualization that work on log.file.path that capture the ingress log count to identify when logs stop flowing in. This has stopped working for 2.3.110 for some reason. And gget the following error:

shards failed:
"Text fields are not optimised for operations that require per-document field data like aggregations and sorting, so these operations are disabled by default. Please use a keyword field instead. Alternatively, set fielddata=true on [log.file.path] in order to load field data by uninverting the inverted index. Note that this can use significant memory."

Is this an issue with the update?

**Comment by weslambert** [comment]
It seems like the data type for this field could have changed since versions.  We introduced a bunch of different changes in 2.3.110 to work towards ECS data type compliance. Does this field show as a conflicts if you navigate to Kibana -> Stack Management -> Index Patterns?

**Comment by mciantar** [comment]
No it does not show as a Conflict.

**Comment by weslambert** [comment]
What does the mapping look like if you go to `Kibana -> Stack Magement -> Index Patterns -> so-*`, then search for the field?
